src/main/web_src/tsconfig.json

```diff
@@ -12,7 +12,8 @@
 		"lib": ["esnext", "dom"],
 		"baseUrl": ".",
 		"paths": {
-			"~/*": ["./src/*"]
+			"~/*": ["./src/*"],
+			"@/*": ["./src/*"]
```

Comment:
Hvorfor ha både ~ og @ som samme path?

Reply:
Fordi jeg har lyst til å foreslå å bytte til @.

Synes det er finere å si AT Api (@/api) eller AT Compoents (@/components). Hva synes dere? Det er typescript pakkene gjør

Reply:
Høres ok ut for meg.

Reply:
@ingvsto-nav går det bra for deg?

Reply:
Hadde oversett denne kommentaren, men ja, et bytte høres bra ut 😄